### DISCUSSION TITLE
Feature Request:  investigate if xlsx files holding grading criteria metadata could be stored as JSON files

### DISCUSSION THREAD

**Discussion by millerg23** [pharmaverse/admiral#2607] [Ideas]
### Feature Idea

Currently its not easy to see changes to the xlsx metadata files holding the lab grading. Ideally they could be stored as JSON files, so developers can rtreack changes, investigation required that JSON files would be easy to read and of course update.

**Comment by bms63** [comment]
@millerg23 i feel like this is more of a discussion topic - which I am keen on seeing happen!! but feels like this issue is too vaugue.

Going to move over to Discussions.

**Comment by manciniedoardo** [comment]
Closing as this is now done
